frontend/router/index.tsx

```diff
@@ -120,6 +121,11 @@ const routes = (
               <Route path="options" component={AgentOptionsPage} />
             </Route>
           </Route>
+          <Route path="labels">
+            <IndexRedirect to={"manage"} />
```

Comment:
im thinking we'd want to redirect to `labels/new` from `labels`. what do you think?